src/EventProcessor/EventProcessorInterface.js

```diff
@@ -1,5 +1,5 @@
-goog.require('measurementlibrary.Storage.StorageInterface');
-goog.module('measurementlibrary.EventProcessor.EventProcessorInterface');
+goog.require('measurement_library.Storage.StorageInterface');
```

Comment:
Nit: Conventionally, we use camelCase for module names. I don't think it's specified anywhere in the style guide, but here it is for your reference: https://google.github.io/styleguide/jsguide.html#file-goog-module

Reply:
I might be misunderstanding something, but this seems to conflict with what you said earlier
https://github.com/google/data-layer-helper/pull/53#discussion_r460156625